css/styles.css

```diff
@@ -0,0 +1,18876 @@
+}
+.DateRangePicker_picker_6.DateRangePicker_picker_6.DateRangePicker_picker_6.DateRangePicker_picker_6.DateRangePicker_picker_6.DateRangePicker_picker_6 {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker_7.DateRangePicker_picker_7.DateRangePicker_picker_7.DateRangePicker_picker_7.DateRangePicker_picker_7.DateRangePicker_picker_7.DateRangePicker_picker_7 {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker_8.DateRangePicker_picker_8.DateRangePicker_picker_8.DateRangePicker_picker_8.DateRangePicker_picker_8.DateRangePicker_picker_8.DateRangePicker_picker_8.DateRangePicker_picker_8 {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9.DateRangePicker_picker_9 {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10.DateRangePicker_picker_10 {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11.DateRangePicker_picker_11 {
+  z-index: 1;
+  background-color: #fff;
+  position: absolute
+}
+.DateRangePicker_picker__rtl_1 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_2.DateRangePicker_picker__rtl_2 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_3.DateRangePicker_picker__rtl_3.DateRangePicker_picker__rtl_3 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_4.DateRangePicker_picker__rtl_4.DateRangePicker_picker__rtl_4.DateRangePicker_picker__rtl_4 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_5.DateRangePicker_picker__rtl_5.DateRangePicker_picker__rtl_5.DateRangePicker_picker__rtl_5.DateRangePicker_picker__rtl_5 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_6.DateRangePicker_picker__rtl_6.DateRangePicker_picker__rtl_6.DateRangePicker_picker__rtl_6.DateRangePicker_picker__rtl_6.DateRangePicker_picker__rtl_6 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_7.DateRangePicker_picker__rtl_7.DateRangePicker_picker__rtl_7.DateRangePicker_picker__rtl_7.DateRangePicker_picker__rtl_7.DateRangePicker_picker__rtl_7.DateRangePicker_picker__rtl_7 {
+  direction: rtl
+}
+.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8.DateRangePicker_picker__rtl_8 {
```

Comment:
also it seems like all of these should be collapsed - ie, `a { blah } b { blah }` should always be `a, b { blah }`

Reply:
again this is due partly to the autogeneration of this file and i need to... probably clean it up a bit more/read more into clean-css options.

Reply:
the output of the css file is totes a nonblocker, but it'd be nice to file issues somewhere to document these kinds of things